Cargo.toml

```diff
@@ -11,14 +11,21 @@ documentation = "https://docs.rs/wgpu_glyph"
 readme = "README.md"
 
 [dependencies]
-wgpu = "0.7"
 glyph_brush = "0.7"
 log = "0.4"
 
 [dependencies.bytemuck]
 version = "1.4"
 features = ["derive"]
 
+[dependencies.wgpu]
+version = "0.8"
+# NB: wgpu switched the default shader translator to `naga` in in 0.8.0,
+# but it currently produces syntax errors when translating this crate's shaders
+# to GLSL for `gfx_backend_gl`.
+# So, for now, tell it to use the legacy `spirv_cross` shader translator.
+features = ["cross"] 
+
```

Comment:
Left a comment here describing the issue, and here is the message from the fatal error I received if `wgpu/cross` was not enabled:

```
[2021-05-02T00:22:17Z ERROR gfx_backend_gl] [ShaderCompiler/Error] ID 1 : 0:10(2): error: syntax error, unexpected NEW_IDENTIFIER
wgpu error: Validation Error

Caused by:
    In Device::create_render_pipeline
    Internal error in VERTEX shader: Shader module failed to compile: 0:10(2): error: syntax error, unexpected NEW_IDENTIFIER
 shader compilation failed: Vertex


thread 'main' panicked at 'Handling wgpu errors as fatal by default', /home/adam/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.8.0/src/backend/direct.rs:1955:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
If the issue is only present when using OpenGL, then it may be okay to switch to `naga`. As far as I know, the OpenGL backend is very experimental.